domain/src/main/java/com/aconno/acnsensa/domain/model/readings/AccelerometerReading.kt

```diff
@@ -0,0 +1,15 @@
+package com.aconno.acnsensa.domain.model.readings
+
+import com.aconno.acnsensa.domain.model.SensorType
+
+class AccelerometerReading(
+    timestamp: Long,
+    val accelerometerX: Number,
```

Comment:
Accelerometer is already in the class name, so x or xAxis is enough for property name.

Reply:
Good point!